assets/js/domain-verifier-checker.js

```diff
@@ -0,0 +1,179 @@
+//Browserify was used to create domain-verifier-bundle.js so that we can use 'require' in the browser
```

Comment:
I'm not exactly a fan of shipping a 1.3mb javascript file (`domain-verifier-bundle.js`) so maybe we can look into a way to cut that back. Even ripple-lib, which I think depends on these same requirements and then some, only weighs in at like 700kb.